src/plugin.json

```diff
@@ -7,7 +7,7 @@
   "autoEnabled": true,
   "info": {
     "keywords": ["app", "tempo", "traces", "explore"],
-    "description": "Grafana app plugin that allows users for a query-less way to navigate and visualize trace data stored in Tempo.",
+    "description": "Grafana app plugin that provides a queryless way for users to navigate and visualize trace data stored in Tempo",
```

Comment:
I didn't want to change this too much, but if description was `Queryless exploration of trace data stored in Tempo`, then it would match metrics and logs more. :)

Reply:
Also `Queryless exploration of trace data stored in Tempo` is in package.json